src/models/TasksList.ts

```diff
@@ -0,0 +1,9 @@
+import Card from './Card'
+import Task from './Task'
+
+export default interface Attachment {
+    name: string,
+    pos: number,
+    card: Card,
+    tasks: Task[]
```

Comment:
I will consider this to be the taskList. 

The double Link between task and taskList seems weird to me. Are you sure about that ?